sherpa-onnx/java/app/src/main/java/com/k2fsa/sherpaonnx/rcglib/WavFile.java

```diff
@@ -0,0 +1,538 @@
+// Wav reader 
```

Comment:
Could you wrap
https://github.com/k2-fsa/sherpa-onnx/blob/ae1f9e791459211e8fee25a8d9b6c575d8577d35/sherpa-onnx/jni/jni.cc#L295
to java and use it to replace this file?